product-service/repository/index.ts

```diff
@@ -1,9 +1,71 @@
-import { products } from './productList';
+import { Client, ClientConfig } from 'pg';
+import { config } from "./config";
+import { Product } from "../types";
 
-export function getById(id: string) {
-   return Promise.resolve(products.find(({id: productId}) => productId === id));
-}
+const GET_ALL = 'SELECT * FROM products INNER JOIN stocks ON stocks.product_id = products.id';
+const GET_BY_ID = `${GET_ALL} WHERE products.id = $1`;
+const CREATE_ITEM = 'INSERT INTO products (title, description, author, price) VALUES ($1, $2, $3, $4) RETURNING id';
+const ADD_TO_STOCK = 'INSERT INTO stocks (product_id, count) VALUES ($1, $2)';
+
+export class ProductRepository {
+    private readonly config: ClientConfig;
+
+    constructor(config: ClientConfig) {
+        this.config = config;
+    }
+
+    private async connect() {
+        const client = new Client(this.config);
+        await client.connect();
+
+        return client;
+    }
+
+    async getAll() {
```

Comment:
any types?

Reply:
Not any

![image](https://user-images.githubusercontent.com/7376989/98700262-724fe200-2391-11eb-8621-5a663be3248c.png)

Reply:
Added return types